docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -225,10 +225,6 @@ For instance, if you have a one-year license (default), you are not licensed to
 
 ## Support
 
-For crowdsourced technical questions from expert MUI devs in our community. Also visited by the MUI Core team.
-
-[Post a question](https://stackoverflow.com/questions/tagged/mui)
```

Comment:
This had nothing to do here, it was my mistake.

Reply:
Where is this information going?
I'm thinking on the discussion we had on perceiving an increase of questions on our GitHub issues, and I think we still don't have a clear answer for it.

Reply:
This was a duplication of content. See the updated content https://deploy-preview-3530--material-ui-x.netlify.app/components/data-grid/getting-started/#support